src/chains.ts

```diff
@@ -53,4 +53,36 @@ export const chains: ChainData[] = [
     explorer: "",
     exchangeRate: 1,
   },
+  {
+    name: "Polygon zkEVM Testnet",
+    symbol: "polyETH",
+    url: "https://rpc.public.zkevm-test.net",
+    chainID: 1442n,
+    explorer: "https://mumbai.polygonscan.com/",
```

Comment:
Is this the correct explorer? I think the mumbia test network is a PoS test chain which is different from the zkevm L2